charmcraft/commands/init.py

```diff
@@ -126,6 +129,8 @@ def run(self, args):
             template_name = template_name[:-3]
             logger.debug("Rendering %s", template_name)
             path = args.path / template_name
+            if path.exists():
```

Comment:
What if it's a directory?

If I have an empty "src" directory, this code will not create the "charm.py" in it. Should it?

Reply:
I don't understand why this would not work for a directory. path.exists would be true and we would continue. Then when we got to the file it would write it

Reply:
@facundobatista

Reply:
Ah, now I understand the code: it **cannot** be a directory, as `path` is always the full path of the template file. So we're ok.